services/terraformer/templates/hetzner.tpl

```diff
@@ -85,7 +85,30 @@ resource "hcloud_server" "{{ $node.Name }}" {
     "managed-by"      : "Claudie"
     "claudie-cluster" : "{{ $clusterName }}-{{ $clusterHash }}"
   }
+
+{{- if not $nodepool.IsControl }}
+  user_data = <<-EOF
+#!/bin/bash
```

Comment:
should we add `set -euxo pipefail`?

we don't want the script to end even with errors, do we?
not sure what difference a script failure would make here tho, we'd need to find out how errors in `user_data` are handled by Hetzner here...

but I definitely wouldn't want the script to modify fstab if it could not mount the disk in the previous step, much less format it..that modification would in fact result in at least the machine not being able to boot again (unable to mount disk).

also, shoudln't we run `systemctl daemon-reload` after modifying fstab to let the system know there's a new mount?